### PR TITLE
PRIME-1377 Add explanatory text to Step 2, Next Steps to get PharmaNet

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

**Review by neophyte57 — APPROVED** [review]

**Review by AnaisHebert — APPROVED** [review]
🎈

File: prime-angular-frontend/src/app/modules/enrolment/shared/components/next-steps-infographic/next-steps-infographic.component.html [hunk 1]
```diff
@@ -29,10 +34,10 @@ <h5>
     </h5>
     <p>
       <button mat-flat-button
-        type="button"
-        color="primary"
-        *ngIf="hasVCIssuance"
-        (click)="openQR($event)">
+              type="button"
```
Comment (itchison) [inline prime-angular-frontend/src/app/modules/enrolment/shared/components/next-steps-infographic/next-steps-infographic.component.html:37]: Just curious why this change occurred? Is there a new linter in play?

Comment (mtpultz) [reply prime-angular-frontend/src/app/modules/enrolment/shared/components/next-steps-infographic/next-steps-infographic.component.html:37]: No someone doesn't have the linter turned on in VSCode.  The HTML entity attributes should always align vertically except on elements that aren't block-level like `<a href="...">...</a>`
